src/core/runtime/runtime.cc

```diff
@@ -118,6 +119,17 @@ static ReturnValues extract_scalar_task(const Task* task,
                    exec_proc.id);
 }
 
+/*static*/ void Core::report_unexpected_exception(const char* task_name,
+                                                  const legate::TaskException& e)
+{
+  log_legate.error(
+    "Task %s threw an exception \"%s\", but the task did not declare any exception. "
+    "Please specify a Python exception that you want this exception to be re-thrown with "
+    "using 'throws_exception'.",
+    task_name,
+    e.error_message().c_str());
+}
```

Comment:
Shouldn't we also `LEGATE_ABORT` at this point?

Reply:
`Logger::error` kills the process, so no need to add `LEGATE_ABORT`.

Reply:
I see. In that case the `abort()` statement in the definition of `LEGATE_ABORT` is extraneous.
https://github.com/nv-legate/legate.core/blob/branch-22.07/src/legate_defines.h#L23

Reply:
@manopapad I was wrong about the logger and `Logger::error` actually doesn't kill the process, so I added `LEGATE_ABORT`: c8abdf7065679ce9f6e79d5b32e2dbbd2c96af99